docs: update alert list visualization (#87815)
grafana/grafana: docs/sources/panels-visualizations/visualizations/alert-list/index.md

    - inactive: 5
  - **Time (asc) -** Newest active alert instances first.
  - **Time (desc) -** Oldest active alert instances first.

### Alerts linked to this dashboard 

Shows alerts only from the dashboard the alert list is in.

## Filter

These options allow you to limit alerts shown to only those that match the query, folder, or tags you choose.

### Alert name

Enter an alert name query.

### Alert instance label

> Filter alert instances using [label](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/) querying, ex: `{severity="critical", instance=~"cluster-us-.+"}`.

Reviewer: ```suggestion
Filter alert instances using [label](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/) querying. For example,`{severity="critical", instance=~"cluster-us-.+"}`.
```

### Datasource

Filter alerts from the selected data source.

### Folder

> Select a folder. Only alerts from dashboards in the folder selected will be displayed.

Reviewer: ```suggestion
Filter alerts by the selected folder. Only alerts from dashboards in this folder are displayed.
```
